lib/README.md

```diff
@@ -0,0 +1,80 @@
+# Lib Package
```

Comment:
How about linking to https://godoc.org/github.com/opencontrol/compliance-masonry/lib (somewhere)?